src/franka_driver.cc

```diff
@@ -9,11 +9,35 @@ namespace dru = utils;
 
 namespace franka_driver {
 
-int do_main(std::string param_yaml) {
+int do_main(std::string param_yaml, uint verbosity) {
   dexai::create_log("franka_driver");
-  int verbose = 0;
+
+  switch (verbosity) {
```

Comment:
this is a good intern project to go clean-up the code to be consistent everywhere...